### ISSUE TITLE
sanitycheck for qemu_cortex_r5 fails 

### ISSUE THREAD

**Issue by andrewboie** [zephyrproject-rtos/zephyr#22036]
```
Deleting output directory /dev/shm/sanity-out
INFO    - JOBS: 48
INFO    - Building initial testcase list...
INFO    - 87 test configurations selected, 647 configurations discarded due to filters.
INFO    - Adding tasks to the queue...
INFO    - Total complete:   24/  87  27%  skipped:   22, failed:    0
ERROR   - qemu_cortex_r5            samples/philosophers/sample.kernel.philosopher.semaphores FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/samples/philosophers/sample.kernel.philosopher.semaphores/build.log
INFO    - Total complete:   25/  87  28%  skipped:   22, failed:    1
ERROR   - qemu_cortex_r5            samples/philosophers/sample.kernel.philosopher     FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/samples/philosophers/sample.kernel.philosopher/build.log
INFO    - Total complete:   26/  87  29%  skipped:   22, failed:    2
ERROR   - qemu_cortex_r5            samples/philosophers/sample.kernel.philosopher.same_prio FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/samples/philosophers/sample.kernel.philosopher.same_prio/build.log
INFO    - Total complete:   27/  87  31%  skipped:   22, failed:    3
ERROR   - qemu_cortex_r5            samples/philosophers/sample.kernel.philosopher.static FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/samples/philosophers/sample.kernel.philosopher.static/build.log
INFO    - Total complete:   28/  87  32%  skipped:   22, failed:    4
ERROR   - qemu_cortex_r5            samples/cpp_synchronization/sample.cpp.synchronization FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/samples/cpp_synchronization/sample.cpp.synchronization/build.log
INFO    - Total complete:   29/  87  33%  skipped:   22, failed:    5
ERROR   - qemu_cortex_r5            samples/philosophers/sample.kernel.philosopher.coop_only FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/samples/philosophers/sample.kernel.philosopher.coop_only/build.log
INFO    - Total complete:   31/  87  35%  skipped:   22, failed:    6
ERROR   - qemu_cortex_r5            samples/philosophers/sample.kernel.philosopher.fifos FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/samples/philosophers/sample.kernel.philosopher.fifos/build.log
INFO    - Total complete:   32/  87  36%  skipped:   22, failed:    7
ERROR   - qemu_cortex_r5            samples/philosophers/sample.kernel.philosopher.stacks FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/samples/philosophers/sample.kernel.philosopher.stacks/build.log
INFO    - Total complete:   33/  87  37%  skipped:   22, failed:    8
ERROR   - qemu_cortex_r5            samples/subsys/logging/logger/sample.logger        FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/samples/subsys/logging/logger/sample.logger/build.log
INFO    - Total complete:   34/  87  39%  skipped:   22, failed:    9
ERROR   - qemu_cortex_r5            samples/philosophers/sample.kernel.philosopher.preempt_only FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/samples/philosophers/sample.kernel.philosopher.preempt_only/build.log
INFO    - Total complete:   35/  87  40%  skipped:   22, failed:   10
ERROR   - qemu_cortex_r5            tests/subsys/logging/log_output/logging.log_output FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/subsys/logging/log_output/logging.log_output/build.log
INFO    - Total complete:   36/  87  41%  skipped:   22, failed:   11
ERROR   - qemu_cortex_r5            samples/philosophers/sample.kernel.philosopher.tracing FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/samples/philosophers/sample.kernel.philosopher.tracing/build.log
INFO    - Total complete:   37/  87  42%  skipped:   22, failed:   12
ERROR   - qemu_cortex_r5            tests/subsys/debug/tracing/tracing.backends.openocd FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/subsys/debug/tracing/tracing.backends.openocd/build.log
INFO    - Total complete:   38/  87  43%  skipped:   22, failed:   13
ERROR   - qemu_cortex_r5            samples/philosophers/sample.kernel.philosopher.lifos FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/samples/philosophers/sample.kernel.philosopher.lifos/build.log
INFO    - Total complete:   39/  87  44%  skipped:   22, failed:   14
ERROR   - qemu_cortex_r5            samples/drivers/crypto/sample.drivers.crypto.mbedtls.micro FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/samples/drivers/crypto/sample.drivers.crypto.mbedtls.micro/build.log
INFO    - Total complete:   40/  87  45%  skipped:   22, failed:   15
ERROR   - qemu_cortex_r5            samples/application_development/external_lib/sample.app_dev.external_lib FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/samples/application_development/external_lib/sample.app_dev.external_lib/build.log
INFO    - Total complete:   41/  87  47%  skipped:   22, failed:   16
ERROR   - qemu_cortex_r5            samples/hello_world/sample.basic.helloworld        FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/samples/hello_world/sample.basic.helloworld/build.log
INFO    - Total complete:   42/  87  48%  skipped:   22, failed:   17
ERROR   - qemu_cortex_r5            tests/lib/ringbuffer/libraries.data_structures     FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/lib/ringbuffer/libraries.data_structures/build.log
INFO    - Total complete:   43/  87  49%  skipped:   22, failed:   18
ERROR   - qemu_cortex_r5            tests/subsys/logging/log_list/logging.log_list     FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/subsys/logging/log_list/logging.log_list/build.log
INFO    - Total complete:   44/  87  50%  skipped:   22, failed:   19
ERROR   - qemu_cortex_r5            tests/subsys/logging/log_msg/logging.log_msg       FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/subsys/logging/log_msg/logging.log_msg/build.log
INFO    - Total complete:   45/  87  51%  skipped:   22, failed:   20
ERROR   - qemu_cortex_r5            tests/subsys/logging/log_immediate/logging.log_immediate FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/subsys/logging/log_immediate/logging.log_immediate/build.log
INFO    - Total complete:   46/  87  52%  skipped:   22, failed:   21
ERROR   - qemu_cortex_r5            tests/lib/c_lib/libraries.libc                     FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/lib/c_lib/libraries.libc/build.log
INFO    - Total complete:   47/  87  54%  skipped:   22, failed:   22
ERROR   - qemu_cortex_r5            tests/kernel/timer/timer_monotonic/kernel.timer.monotonic FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/kernel/timer/timer_monotonic/kernel.timer.monotonic/build.log
INFO    - Total complete:   48/  87  55%  skipped:   22, failed:   23
ERROR   - qemu_cortex_r5            tests/ztest/custom_output/testing.ztest.customized_output FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/ztest/custom_output/testing.ztest.customized_output/build.log
INFO    - Total complete:   49/  87  56%  skipped:   22, failed:   24
ERROR   - qemu_cortex_r5            tests/ztest/base/testing.ztest.verbose_2           FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/ztest/base/testing.ztest.verbose_2/build.log
INFO    - Total complete:   50/  87  57%  skipped:   22, failed:   25
ERROR   - qemu_cortex_r5            tests/ztest/custom_output/testing.ztest.regular_output FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/ztest/custom_output/testing.ztest.regular_output/build.log
INFO    - Total complete:   51/  87  58%  skipped:   22, failed:   26
ERROR   - qemu_cortex_r5            tests/ztest/base/testing.ztest.verbose_0           FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/ztest/base/testing.ztest.verbose_0/build.log
INFO    - Total complete:   52/  87  59%  skipped:   22, failed:   27
ERROR   - qemu_cortex_r5            tests/ztest/base/testing.ztest.verbose_1           FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/ztest/base/testing.ztest.verbose_1/build.log
INFO    - Total complete:   53/  87  60%  skipped:   22, failed:   28
ERROR   - qemu_cortex_r5            tests/application_development/gen_inc_file/buildsystem.include_file FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/application_development/gen_inc_file/buildsystem.include_file/build.log
INFO    - Total complete:   54/  87  62%  skipped:   22, failed:   29
ERROR   - qemu_cortex_r5            tests/application_development/libcxx/application_development.cpp.libcxx.exceptions FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/application_development/libcxx/application_development.cpp.libcxx.exceptions/build.log
INFO    - Total complete:   55/  87  63%  skipped:   22, failed:   30
ERROR   - qemu_cortex_r5            tests/application_development/libcxx/application_development.cpp.libcxx FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/application_development/libcxx/application_development.cpp.libcxx/build.log
INFO    - Total complete:   57/  87  65%  skipped:   22, failed:   31
ERROR   - qemu_cortex_r5            tests/net/buf/net.buf                              FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/net/buf/net.buf/build.log
INFO    - Total complete:   58/  87  66%  skipped:   22, failed:   32
ERROR   - qemu_cortex_r5            tests/application_development/cpp/application_development.cpp.main FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/application_development/cpp/application_development.cpp.main/build.log
INFO    - Total complete:   59/  87  67%  skipped:   22, failed:   33
ERROR   - qemu_cortex_r5            tests/lib/fdtable/libraries.os.fdtable             FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/lib/fdtable/libraries.os.fdtable/build.log
INFO    - Total complete:   60/  87  68%  skipped:   22, failed:   34
ERROR   - qemu_cortex_r5            tests/posix/common/portability.posix.common        FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/posix/common/portability.posix.common/build.log
INFO    - Total complete:   61/  87  70%  skipped:   22, failed:   35
ERROR   - qemu_cortex_r5            tests/posix/fs/portability.posix.fs                FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/posix/fs/portability.posix.fs/build.log
INFO    - Total complete:   62/  87  71%  skipped:   22, failed:   36
ERROR   - qemu_cortex_r5            tests/subsys/jwt/libraries.encoding.jwt            FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/subsys/jwt/libraries.encoding.jwt/build.log
INFO    - Total complete:   63/  87  72%  skipped:   22, failed:   37
ERROR   - qemu_cortex_r5            samples/drivers/crypto/sample.drivers.crypto.mbedtls FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/samples/drivers/crypto/sample.drivers.crypto.mbedtls/build.log
INFO    - Total complete:   65/  87  74%  skipped:   22, failed:   38
ERROR   - qemu_cortex_r5            tests/crypto/rand32/crypto.rand32.random_sw_systimer FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/crypto/rand32/crypto.rand32.random_sw_systimer/build.log
INFO    - Total complete:   66/  87  75%  skipped:   22, failed:   39
ERROR   - qemu_cortex_r5            tests/crypto/rand32/crypto.rand32                  FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/crypto/rand32/crypto.rand32/build.log
INFO    - Total complete:   67/  87  77%  skipped:   22, failed:   40
ERROR   - qemu_cortex_r5            tests/drivers/ipm/drivers.ipc.mailbox              FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/drivers/ipm/drivers.ipc.mailbox/build.log
INFO    - Total complete:   69/  87  79%  skipped:   22, failed:   41
ERROR   - qemu_cortex_r5            tests/lib/sprintf/libraries.libc.sprintf           FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/lib/sprintf/libraries.libc.sprintf/build.log
INFO    - Total complete:   70/  87  80%  skipped:   22, failed:   42
ERROR   - qemu_cortex_r5            tests/subsys/logging/log_core/logging.log_core     FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/subsys/logging/log_core/logging.log_core/build.log
INFO    - Total complete:   71/  87  81%  skipped:   22, failed:   43
ERROR   - qemu_cortex_r5            tests/lib/json/libraries.encoding.json             FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/lib/json/libraries.encoding.json/build.log
INFO    - Total complete:   74/  87  85%  skipped:   22, failed:   44
ERROR   - qemu_cortex_r5            tests/subsys/shell/shell_history/shell.history     FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/subsys/shell/shell_history/shell.history/build.log
INFO    - Total complete:   77/  87  88%  skipped:   22, failed:   45
ERROR   - qemu_cortex_r5            tests/shell/shell.core                             FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/shell/shell.core/build.log
INFO    - Total complete:   79/  87  90%  skipped:   22, failed:   46
ERROR   - qemu_cortex_r5            tests/posix/common/portability.posix.common.newlib FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/posix/common/portability.posix.common.newlib/build.log
INFO    - Total complete:   81/  87  93%  skipped:   22, failed:   47
ERROR   - qemu_cortex_r5            tests/posix/fs/portability.posix.fs.newlib         FAILED: Exited with 2
ERROR   - see: /dev/shm/sanity-out/qemu_cortex_r5/tests/posix/fs/portability.posix.fs.newlib/build.log
INFO    - Total complete:   87/  87  100%  skipped:   22, failed:   48
INFO    - 17 of 65 tests passed (26.15%), 48 failed, 22 skipped with 0 warnings in 44.21 seconds
INFO    - In total 745 test cases were executed on 1 out of total 218 platforms (0.46%)
```

**Comment by jhedberg** [comment]
Will be solved with the new SDK

**Comment by jhedberg** [comment]
@galak assigned this to you since it'll be resolved with the new SDK

**Comment by jhedberg** [comment]
@galak has this been resolved (i.e. should it be closed) now that we're using the new SDK?

**Comment by galak** [comment]
appears to be fixed with SDK 0.11.x release.
